### DISCUSSION TITLE
2.2 client not working

### DISCUSSION THREAD

**Discussion by mahierth** [Cvolton/GMDprivateServer#1067] [Q&A]
every time i keep trying to replace boomlings.com with my GDPS server thing it shows up with this
![error](https://github.com/Cvolton/GMDprivateServer/assets/99015735/cf527b54-5c60-4194-94c6-0baf2ce1b41e)

(i am on windows 7)

**Comment by MegaSa1nt** [comment]
Changing links with different characters count

**Reply by mahierth** [reply]
wdym

**Reply by MegaSa1nt** [reply]
Your GDPS link has different characters count

**Reply by mahierth** [reply]
but https://mahiergdps.5v.pl/database (my GDPS link) is 33 characters (including https and database)

**Reply by MegaSa1nt** [reply]
And `https://www.boomlings.com/database` is 34 characters, add additional "/" at the end of your link

**Reply by mahierth** [reply]
still didnt work

**Reply by Kingminer7** [reply]
try renaming the /database folder to something like /databasef and changing that in your app.

**Reply by mahierth** [reply]
1. it showed 57 occurrences

2. it still didn't work

**Reply by MegaSa1nt** [reply]
Did you setup GDPS right?

**Reply by mahierth** [reply]
yes
